src/payment.js

```diff
@@ -357,18 +330,18 @@ Payment.from = function (origin) {
 Payment.updateFees = function () {
   return function (payment) {
     return API.getFees().then(
-          function (fees) {
-            payment.fees = fees;
-            payment.feePerKb = fees.default.fee;
-            return payment;
-          }
-        ).catch(
-          // this could fail for network issues - fallback default fee
-          function (error) {
-            console.log(error);
-            return payment;
-          }
-        );
+      function (fees) {
+        payment.fees = fees;
+        payment.feePerKb = fees.legacyCapped * 1000;
```

Comment:
Instead of multiplying by 1000, could you make a helper function `bytesToKb`? (And use it in other places too)